### PR TITLE
Add cats_eirini suite

### PR DESCRIPTION
**Pull request by jimmykarily**
This PR adds the needed code in order to run a different set of tests for Eirini. It will be used to address this issue: https://github.com/cloudfoundry-incubator/kubecf/issues/1007

It copies its logic from the cats-internetless suite:
- creates a secret with cats configuration
- patches the cats quarksjob with use the new secret
- runs cats
- reverts all changes in the end

### CODE REVIEW

File: modules/tests/kubecf-test.sh [hunk 1]
```diff
@@ -174,6 +174,83 @@ cleanup_cats_internetless() {
...
+
+  qjob="$(get_resource_name qjob "acceptance-tests")"
+  cats_secret_name="$(
+    kubectl get ${qjob} --namespace "${KUBECF_NAMESPACE}" -o json | \
+    jq -r '.spec.template.spec.template.spec.volumes[] | select(.name=="ig-resolved").secret.secretName')"
+
+  cats_secret="$(kubectl get secrets --export -n "${KUBECF_NAMESPACE}" "${cats_secret_name}" -o yaml)"
+  # This is what upstream Eirini was running and was green:
+  # https://github.com/cloudfoundry-incubator/eirini-ci/blob/09dcce6d9e900f693dfc1a6da70b5a526cf7de18/pipelines/dhall-modules/jobs/run-core-cats.dhall#L52-L86
+  suites=$(paste -d',' -s <(cat <<-SUITES
```
Comment (jimmykarily) [inline modules/tests/kubecf-test.sh:188]: Docs: https://github.com/SUSE/cf-acceptance-tests-release/blob/master/jobs/acceptance-tests/spec#L47

File: modules/tests/kubecf-test.sh [hunk 2]
```diff
@@ -174,6 +174,87 @@ cleanup_cats_internetless() {
...
+  qjob="$(get_resource_name qjob "acceptance-tests")"
+  cats_secret_name="$(
+    kubectl get ${qjob} --namespace "${KUBECF_NAMESPACE}" -o json | \
+    jq -r '.spec.template.spec.template.spec.volumes[] | select(.name=="ig-resolved").secret.secretName')"
+
+  cats_secret="$(kubectl get secrets --export -n "${KUBECF_NAMESPACE}" "${cats_secret_name}" -o yaml)"
+  # This is what upstream Eirini was running and was green:
+  # https://github.com/cloudfoundry-incubator/eirini-ci/blob/09dcce6d9e900f693dfc1a6da70b5a526cf7de18/pipelines/dhall-modules/jobs/run-core-cats.dhall#L52-L86
+
+  # Normally, it only makes sense to disable test groups that enabled by default
```
Comment (viccuad) [inline modules/tests/kubecf-test.sh:189]:
```suggestion
  # Normally, it only makes sense to disable test groups that are enabled by default
```

File: modules/tests/kubecf-test.sh [hunk 3]
```diff
@@ -174,6 +174,87 @@ cleanup_cats_internetless() {
...
+
+  updated_volumes=$(echo ${original_volumes} | \
+    jq -r 'map((select(.name=="ig-resolved") | .secret.secretName) |= "cats-eirini")')
+
+  patch='{ "spec": { "template": { "spec": { "template": { "spec": { "volumes": '${updated_volumes}' } } } } } }'
+
+  kubectl patch ${qjob} --namespace "${KUBECF_NAMESPACE}" --type merge --patch "${patch}"
+}
+
+# Re-enables internet access again and mounts the original secret in the qjob
```
Comment (viccuad) [inline modules/tests/kubecf-test.sh:245]: Internet is not disabled at any point for cats-eirini, no?

Comment (jimmykarily) [reply modules/tests/kubecf-test.sh:245]: yes, copy-paste error. Fixing.

**Pull request merged**